components/list_modal.jsx

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This is a new reusable component that we can potentially use to replace all modals that look like this.

<img width="600" alt="Screen Shot 2019-05-02 at 10 52 53 AM" src="https://user-images.githubusercontent.com/1149597/57084524-785b0200-6cc8-11e9-81d0-a1dd7272786c.png">